submission-package.Rmd

```diff
@@ -39,7 +39,7 @@ We assume the paths to the two folders are `esubdemo/` and `ectddemo/` below.
 
 ## The whole game
 
-In eCTD deliverable, the analysis dataset and source code are saved
+In eCTD deliverable, the analysis datasets and source codes are saved
```

Comment:
I'm ok with the dataset -> datasets change but it should be source **code** in this context: https://english.stackexchange.com/questions/20455/